includes/Widgets/StoreContactForm.php

```diff
@@ -77,13 +98,15 @@ function widget( $args, $instance ) {
 
             dokan_get_template_part(
                 'widgets/store-contact-form', '', array(
-					'seller_id'  => $seller_id,
-					'store_info' => $store_info,
-					'username'   => $username,
-					'email'      => $email,
+                    'seller_id'  => $seller_id,
+                    'store_info' => $store_info,
+                    'username'   => $username,
+                    'email'      => $email,
                 )
             );
 
+            wp_localize_script( 'dokan-script', 'google_recaptcha', [ 'recaptcha_sitekey' => $this->recaptcha_site_key ] );
```

Comment:
change handle with  `dokan-google-recaptcha` since there is already a handle for recaptcha, also  move this line under file: includes/Assets.php line: 510